docs/copied-from-beats/command-reference.asciidoc

```diff
@@ -15,30 +15,32 @@
 :global-flags: Also see <<global-flags,Global flags>>.
 
 :deploy-command-short-desc: Deploys the specified function to your serverless environment
+
+ifndef::no_dashboards_apm[]
 :export-command-short-desc: Exports the configuration, index template, or a dashboard to stdout
+endif::no_dashboards_apm[]
+
+ifdef::no_dashboards_apm[]
+:export-command-short-desc: Exports the configuration or index template to stdout
+endif::no_dashboards_apm[]
+
 :help-command-short-desc: Shows help for any command
 :keystore-command-short-desc: Manages the <<keystore,secrets keystore>>
 :modules-command-short-desc: Manages configured modules
 :package-command-short-desc: Packages the configuration and executable into a zip file
 :remove-command-short-desc: Removes the specified function from your serverless environment
 :run-command-short-desc: Runs {beatname_uc}. This command is used by default if you start {beatname_uc} without specifying a command
 
-ifndef::deprecate_dashboard_loading[]
-
 ifdef::has_ml_jobs[]
 :setup-command-short-desc: Sets up the initial environment, including the index template, {kib} dashboards (when available), and machine learning jobs (when available)
 endif::[]
 
-ifndef::has_ml_jobs[]
-:setup-command-short-desc: Sets up the initial environment, including the index template and {kib} dashboards (when available)
-endif::[]
-
-endif::[]
-
-ifdef::deprecate_dashboard_loading[]
-
-:setup-command-short-desc: Sets up the initial environment, including the ES index template and {kib} dashboards (deprecated).
+ifdef::no_dashboards_apm[]
+:setup-command-short-desc: Sets up the initial environment, including the ES index template.
+endif::no_dashboards_apm[]
 
+ifndef::has_ml_jobs,no_dashboards_apm[]
+:setup-command-short-desc: Sets up the initial environment, including the index template and {kib} dashboards (when available)
```

Comment:
the `{kib} dashboards (when available)` part should be removed? (sorry if not, not sure i am getting this right)

Reply:
All good :)
L43 only shows up if `has_ml_jobs` *and* `no_dashboards_apm` are not defined. Since we have defined `no_dashboards_apm`, L39 shows up in our documentation